File: 프로그래머스/unrated/181926. 수 조작하기 1/수 조작하기 1.py
def solution(n, control):
    answer = n
    
    for ch in control:
        if ch == 'w':
            answer += 1
        elif ch == 's':
            answer -= 1
        elif ch == 'd':
            answer += 10
        else:
            answer -= 10
    
    return answer

File: 프로그래머스/unrated/181926. 수 조작하기 1/수 조작하기 1.java
class Solution {
    public int solution(int n, String control) {
        int answer = n;
        
        for (int i=0; i<control.length(); i++) {
            char ch = control.charAt(i);
            
            if (ch == 'w') {
                answer++;
            } else if (ch == 's') {
                answer--;
            } else if (ch == 'a') {
                answer -= 10;
            } else {
                answer += 10;
            }
        }
        
        return answer;
    }
}